### PR TITLE
Feature/player turn

### PR DESCRIPTION
**Pull request by andredavii**
function determining player turns.

### CODE REVIEW

File: source/playerTurn.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def playerTurn(players, current_player, getCard):
+    num_players = len(players)
+    if getCard:
+        return current_player
+    else:
+        next_player = (current_player + 1) % num_players
+        return next_player
```
Comment (RobertWheel) [inline source/playerTurn.py:7]: nice and simple, makes sense easy to follow

Comment (RobertWheel) [inline source/playerTurn.py:7]: nice and simple, makes sense easy to follow

File: source/CardGames.py [hunk 2]
```diff
@@ -196,4 +196,12 @@ def playerTurn(players: list, current_player: str, getCard: bool) -> None:
...
\ No newline at end of file
+      return next_player
+  
+def playerTurn(players, current_player, getCard):
+    num_players = len(players)
+    if getCard:
+        return current_player
+    else:
+        next_player = (current_player + 1) % num_players
+        return next_player
```
Comment (Zwilson1301) [inline source/CardGames.py]: Good Job Andre, your code is super clean and concise.

File: source/playerTurn.py (hunk 1 above)
Comment (andredavii) [reply source/playerTurn.py:7]: Thanks for reviewing Robert.

File: source/CardGames.py (hunk 2 above)
Comment (andredavii) [reply source/CardGames.py]: Thanks for the review Zach!

**Pull request merged**